Lib/shutil.py

```diff
@@ -1030,8 +1030,10 @@ def make_archive(base_name, format, root_dir=None, base_dir=None, verbose=0,
     uses the current owner and group.
     """
     sys.audit("shutil.make_archive", base_name, format, root_dir, base_dir)
+    base_name = os.fsdecode(base_name)
```

Comment:
Isn’t fsdecode about bytes-str decoding, and fspath for pathlike-to-OS-native conversion?